packages/inventory/src/shared/AccessDenied.js

```diff
@@ -1,17 +1,26 @@
-import React, { Fragment } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 import { NotAuthorized } from '@redhat-cloud-services/frontend-components/components/cjs/NotAuthorized';
 
-const AccessDenied = (props) => (
+const AccessDenied = ({ title, description, ...props }) => (
     <NotAuthorized
         {...props}
-        serviceName="Inventory"
-        description={
-            <Fragment>
-                <div>Your organization administrator must grant</div>
-                <div>you inventory access to view your systems.</div>
-            </Fragment>
-        }
+        className="ins-c-inventory__no--access"
+        title={title}
+        description={description}
     />
 );
 
+AccessDenied.propTypes = {
+    title: PropTypes.string,
+    description: PropTypes.node
+};
+
+AccessDenied.defaultProps = {
+    title: 'This application requires Inventory permissions',
+    description: <div>
```

Comment:
Is there a reason why it's needed be wrapped in a div?

Reply:
To properly style the message, it needs to be on 2 lines, similiar to dashboard. The selector here would be too weak to overpower PF's one.